src/main/java/picard/util/BedToIntervalList.java

```diff
@@ -64,9 +64,16 @@
             "     -Strand - Indicates +/- strand for the interval (either + or -) <br /> " +
             "     -Interval name - (Each interval should have a unique name) " +
             "</pre>" +
-            "  <br /><br />" +
-            "This tool requires sequence dictionary file (with \".dict\" extension), which can be created from a reference sequence " +
-            "using Picard's CreateSequenceDictionary tool."+
+            "<br/>" +
+            "This tool requires sequence dictionary, provided with the SEQUENCE_DICTIONARY or SD argument. " +
+            "This argument can be any of:" +
+            "<pre>" +
+            "    - A file with .dict extension generated using Picard's CreateSequenceDictionaryTool</br>" +
+            "    - A <name>.fa or <name>.fasta file with a <name>.dict in the same directory</br>" +
```

Comment:
Avoid angle brackets as they will be misinterpreted by the html parser (even despite the pre blocks, iirc); just use something like name.fa or reference_name.fa, people will understand they need to adapt the name.